src/server/db_slice.cc

```diff
@@ -1485,24 +1433,13 @@ void DbSlice::PerformDeletion(PrimeIterator del_it, ExpireIterator exp_it, Engin
 
   DbTableStats& stats = table->stats;
   const PrimeValue& pv = del_it->second;
-  if (pv.IsExternal()) {
```

Comment:
should we call doc_del_cb_ inside PerformDeletion?

Reply:
@dranikpg I think those callbacks are for search, right? what do you think?

Reply:
@dranikpg friendly ping :)

Reply:
Oh sorry 😅 When you call the callback, the data should be still available, no other restrictions are placed